integration-tests/debezium/src/test/java/org/apache/camel/quarkus/component/debezium/common/it/sqlserver/DebeziumSqlserverTest.java

```diff
@@ -89,15 +89,25 @@ protected String getCompanyTableName() {
     @Order(0)
     @EnabledIfSystemProperty(named = PROPERTY_JDBC, matches = ".*")
     public void testReceiveInitCompany() {
-        //receive first record (operation r) for the init company - using larger timeout
-        Response response = receiveResponse("/receiveAsRecord");
+        int i = 0;
 
-        response.then()
-                .statusCode(200);
+        while (i++ < AbstractDebeziumTest.REPEAT_COUNT) {
```

Comment:
We tend to use Awaitility in situations like these so you may consider using it too on the next occasion.